x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_table.tsx

```diff
@@ -77,23 +83,36 @@ export const EngineTable: ReactFC<IEngineTableProps> = ({
     },
     {
       field: 'document_count',
-      name: 'Document Count',
+      name: i18n.translate('xpack.appSearch.enginesOverview.table.column.documentCount', {
+        defaultMessage: 'Document Count',
+      }),
       dataType: 'number',
       render: number => number.toLocaleString(), // Display with comma thousand separators
```

Comment:
Same here, I think react-intl has a `formatNumber` helper...

Reply:
Yup, https://formatjs.io/docs/react-intl/components/#formattednumber does all the hard work for me!